### DISCUSSION TITLE
是否可以加入对Azure openai service的支持

### DISCUSSION THREAD

**Discussion by LucidDreamLab** [immersive-translate/immersive-translate#523] [Ideas]
### 新功能 | New Feature

支持Azure openai service，可以使用Azure的终结点与api key访问gpt

### 为什么您想要添加这些功能 | Why you want to add these features

因为openai的key在无代理的情况下使用风险较大，而azure不会

**Comment by theowenyoung** [comment]
已经支持的哈，见文档： https://immersive-translate.owenyoung.com/services/azure

**Reply by LucidDreamLab** [reply]
但是Azure翻译和azure openai service好像不是同一个，使用Azure openai service是调用gpt来进行翻译

**Reply by theowenyoung** [reply]
> 但是Azure翻译和azure openai service好像不是同一个，使用Azure openai service是调用gpt来进行翻译

sorry，给错链接了，两个都支持的：

https://immersive-translate.owenyoung.com/services/openai#azure-openai

**Reply by LucidDreamLab** [reply]
好滴感谢
